js/blocks/components/advanced-controls.js

```diff
@@ -25,12 +25,12 @@ const AdvancedControls = ( { block } ) => {
 
 	return (
 		<InspectorAdvancedControls key={ `inspector-advanced-controls-${ block.name }` }>
-			<DotTip tipId="block-lab/additional-css-class">
+			<Guide tipId="block-lab/additional-css-class">
 				<p className="bl-dot-tip" dangerouslySetInnerHTML={ { __html: tip } }></p>
 				<p className="bl-dot-tip read-more">
 					<a href="https://getblocklab.com/docs/faqs/" target="_blank" rel="noopener noreferrer">{ __( 'Read more', 'block-lab' ) }</a>
 				</p>
-			</DotTip>
+			</Guide>
```

Comment:
This is one of the reasons that this PR now imports modules, like `@wordpress/components`, instead of using something like:

```js
const { Guide } = wp.components;
```

The `nux` package was removed, and the `DotTip` was moved to the `components` package. So we can control which packages we're using, and don't need conditionals like:

```js
const { DotTip } = wp.nux ? wp.nux.DotTip : wp.components.Guide;
```

Reply:
Also, there used to be deprecation notices of components, which we would have to deal with at some point.